### PR TITLE
fix: add missing showcases images

### LINKED ISSUE #492: discuss: add default text or image for company logo
**Discuss description**
add default text or image for company logo

**Screenshots of the issue**

### PR DESCRIPTION
**Pull request by foolwc**
Fixes: #492

Changes:
+ Add logo for companies:  `vivo`、`金山办公`、`上海泽怡信息科技`、`北京新片场传媒股份有限公司` ;
+ Replace logo for company: `会课` , because it was just like a blank image;  

Screenshots of the change:
Snipaste_2021-08-24_19-23-15

### CODE REVIEW

**Review by KishaniKandasamy — APPROVED** [review]

File: website/static/img/showcase/vivo.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<svg width="104px" height="28px" viewBox="0 0 103 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="海外一期" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (juzhiyuan) [inline website/static/img/showcase/vivo.svg]: What's this?

Comment (foolwc) [reply website/static/img/showcase/vivo.svg]: I copied this from [http://vivo.com/fr](http://vivo.com/fr). Fixed, thanks.

**Review by juzhiyuan — APPROVED** [review]

File: website/static/img/showcase/vivo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...63.506286 0c45.622857 0 55.04 7.789714 72.630857 33.901714 17.590857 26.130286 322.377143 491.428571 322.377143 491.428572 9.728 14.299429 20.041143 20.096 32.859429 20.096 12.836571 0 23.149714-5.796571 32.877714-20.114286 0 0 304.768-465.28 322.377143-491.410286 17.590857-26.112 27.008-33.901714 72.649143-33.901714h106.550857c27.373714 0 35.748571 19.181714 20.315428 42.605714L753.207223 904.100571c-56.502857 83.090286-97.956571 116.681143-190.774857 116.681143-92.8 0-134.272-33.590857-190.738286-116.662857L7.680366 351.378286c-15.433143-23.405714-7.076571-42.587429 20.297143-42.587429z m3319.826286 149.577143h-328.758857c-184.795429 0-201.234286 151.570286-201.234286 197.869714s16.438857 197.851429 201.234286 197.851429h328.758857c184.813714 0 201.252571-151.552 201.252571-197.851429v-1.737143c-0.438857-48.237714-18.706286-196.132571-201.252571-196.132571zM1394.450651 37.010286l56.118858 56.100571c26.24 26.258286 26.24 47.067429 0 73.325714l-56.118858 56.082286c-26.24 26.258286-47.049143 26.258286-73.289142 0l-56.100572-56.100571c-26.258286-26.24-26.258286-47.049143 0-73.307429l56.100572-56.100571c26.24-26.24 47.049143-26.24 73.289142 0z" fill="#242933" p-id="2085"></path></svg>
```
Comment (liuxiran) [inline website/static/img/showcase/vivo.svg]: need new line here

Comment (foolwc) [reply website/static/img/showcase/vivo.svg]: > need new line here

Fixed. Thanks

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Comment by liuxiran** [comment]
merged, @foolwc thanks a lot

**Pull request merged**